python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -363,13 +363,22 @@ def indices(self):
         >>> df.groupby(by=["a"]).indices
         {10: array([0, 1]), 40: array([2])}
         """
-        group_names, offsets, _, grouped_values = self._grouped()
+        offsets, group_keys, (indices,) = self._groupby.groups(
+            [
+                cudf.core.column.as_column(
+                    range(len(self.obj)), dtype=size_type_dtype
+                )
```

Comment:
```suggestion
                cudf.core.column.arange(len(self.obj), dtype=size_type_dtype)
```

Reply:
Not since https://github.com/rapidsai/cudf/pull/14689

Reply:
Weird. I thought there was some changes around `arange` but I didn't recall what had changed. I saw it was still there, but it appears I was clicking through your fork of cudf.